### ISSUE TITLE
No annotations in the action.

### ISSUE THREAD

**Issue by Foxhoundn** [jamesacarr/jest-reporter-github-actions#4]
Hi,

I am not sure if I am doing everything correctly, but I have no annotations when the action finishes.

Jest yarn command: `"test": "./node_modules/.bin/jest --testLocationInResults --reporters=default --reporter=@jamesacarr/jest-reporter-github-actions",`

The test result:
```

    expect(received).toBe(expected) // Object.is equality

    Expected: 1
    Received: 0

      51 |         CharacterStatsMax.HEALTH
      52 |       );
    > 53 |       expect(player.getStat(ResourceTypes.SHIELD)).toBe(1);
         |                                                    ^
      54 |       expect(player.getStat(ResourceTypes.POWER)).toBe(CharacterStatsMax.POWER);
      55 |       expect(player.getGameRoomId()).toBe(1);
      56 |       expect(player.pivot.position).toEqual({ x: 0, z: -19, y: 0 });

      at Object.test (../client/tests/suite.spec.ts:53:52)

PASS ../client/tests/player/player.spec.ts

Test Suites: 1 failed, 1 passed, 2 total
Tests:       1 failed, 6 passed, 7 total
Snapshots:   0 total
Time:        64.768 s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

Github annotations: 
```
Check failure on line 1 in .github

@github-actions
github-actions
/ test

.github#L1
Process completed with exit code 1.
```

Thanks for any info!

**Comment by Foxhoundn** [comment]
Okay there is a typo in the readme `reporter` instead of `reporters`.

Fixing that added the `Jest annotations` but there seems to be another bug.

```
Test Suites: 1 failed, 1 passed, 2 total
Tests:       1 failed, 6 passed, 7 total
Snapshots:   0 total
Time:        57.149 s
Ran all test suites.
Jest Annotations
  TypeError: failedSuites.flatMap is not a function
      at GitHubActionsReporter.testsWithFailures (/home/runner/work/synthax/synthax/src/js/server/node_modules/@jamesacarr/jest-reporter-github-actions/dist/index.js:16:29)
      at GitHubActionsReporter.onRunComplete (/home/runner/work/synthax/synthax/src/js/server/node_modules/@jamesacarr/jest-reporter-github-actions/dist/index.js:7:34)
      at ReporterDispatcher.onRunComplete (/home/runner/work/synthax/synthax/src/js/server/node_modules/@jest/core/build/ReporterDispatcher.js:86:24)
  Error: Process completed with exit code 1.
```

**Comment by jamesacarr** [comment]
Thanks for catching that @Foxhoundn. I'll fix up the README to have the correct info

**Comment by jamesacarr** [comment]
Resolved in dc983a6351d8037959c0b371bdc1c9293745c7a5

**Comment by jamesacarr** [comment]
@Foxhoundn sorry, I completely missed the second part of your comment about the `flatMap` issue. It looks like your tests might not be running on a version of `node` that has `flatMap` (I think it was added in `node` v11?). I'll see if I can swap it out for something that's a bit more backwards-compatible.

**Comment by jamesacarr** [comment]
Should now be resolved in 998089a
